nx-X11/config/cf/lnxLib.rules

```diff
@@ -9,7 +9,7 @@
 #endif
 
 #ifndef UseRpath
-#define UseRpath NO
+#define UseRpath YES
```

Comment:
And that... could fail spectacularly.

It looks as if the RPATH is set conditionally based upon whether `UseInstalled` is defined or not. If it is, `USRLIBDIRPATH` is being used (which is kind-of what we want), otherwise `BUILDLIBDIR`.

I don't see `UseInstalled` being defined anywhere, so I guess we'll be ending up with `BUILDLIBDIR` being used, which is set to `$(TOP)/exports/lib`.

So do we **actually** get stuff RPATH'd to the correct location, or will `nxagent` just fail to execute? Did you test that?

That being said, even IF `USRLIBDIRPATH` is being used, I suspect that it could potentially be set to `/usr/lib/nx-X11` as defined in `nx-X11/config/cf/Imake.tmpl`, overriding the overridden value. If not (for instance because values passed directly to make normally override all variable definitions inside of a Makefile - but I don't know how imake behaves in this respect), we might be in the clear.